### ISSUE TITLE
ArcFace baseline model used in SlerpFace paper

### ISSUE THREAD

**Issue by donghan22** [Tencent/TFace#157]
Hi,

Thanks for your great work.

As stated in paper:
""We compare SlerpFace with an unprotected baseline and five FTP methods: **ArcFace** as the baseline; ""

Could you guide me where can I find this pre-trained model of **ArcFace** model that you used as baseline model in evaluation? @zzzweakman

**Comment by zzzweakman** [comment]
Dear @DongHan9722,

Thank you for your interest in our work. I have uploaded the ArcFace pretrain model to the following [link](https://drive.google.com/drive/folders/1h-yO2h3dyc8DIIZAA9rdgwZ8qcPjDy7P). You can access the model weights and training logs there. I hope you find this helpful.

Best regards,
The SlerpFace Team

**Comment by DongHan9722** [comment]
Hi,

Thanks for your quick replying.

May I ask what FR model you were using to test other FTP methods?




Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by zzzweakman** [comment]
> > Hi,
> > Thanks for your quick replying.
> > May I ask what FR model you were using to test other FTP methods?
> > Sent from Outlook for iOShttps://aka.ms/o0ukef
> > […](#)
> 
> Thank you for your message. I used the model trained with the ArcFace loss on the MS1M dataset. This is the same model that I have just uploaded. By the way, you'd better use TFace to run inference.
